### Selected paper
Wallace, Jason G., et al. "Association mapping across numerous traits reveals patterns of functional variation in maize." PLoS genetics 10.12 (2014): e1004845.

### Introduction
In the [Wallele et al. 2014](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004845#pgen.1004845.s008) paper, the authors conducted Genome-wide association study (GWAS) on 41 metabolic and developmental pheontypes, using the maize Nested Association Mapping (NAM) population. With all the trait associated SNPs (TAS) and candidate genes within the TAS loci, the authors were able to explore the patterns of these TAS and candidate genes, such as distribution, effect size, minor allele frequency, expression etc. 

### Goal of the project
In this project, we tried to repeat **four analysis** of our interests among all those in the original paper.

1. Number of GWAS hits for every trait;
2. Relationship of minor allele frequencies (MAF) and effect size of all the TAS;
3. Expressioin pattern of candidate genes identified in GWAS;
4. Enrichment of candidate genes in the two maize sub-genomes

### Technical details

#### Original paper
##### Phenotype data
Best linear unbiased predictors (BLUPs) for each line within each trait were calculated using [ASReml](version 2.0; http://www.vsni.co.uk/software/asreml). 

##### Genotype data
The input SNPs were a union of all SNPs in [Hapmap1 and Hapmap2](https://www.panzea.org/genotypes), in total ~28.9 million SNPs and and ∼800,000 copy-number variants (CNVs).

##### GWAS model
In the majority of cases phenotypic data had already been processed by fitting a joint-linkage model. 

Forward-regression genome-wide association was then performed with the NamGwasPlugin in [TASSEL](http://www.maizegenetics.net/tassel). This plugin was created specifically to run stepwise forward regression on the Maize NAM population, and takes as input the chromosome-specific residuals, a genetic map, anchor genotypes in the progeny, and founder genotypes to be imputed. Each chromosome was analyzed separately for each phenotype via 100 forward-regression iterations, each of which excluded a random 20% of NAM lines to destabilize spurious associations.

##### Gene Expression data
Maize gene atlas published in [Sekhon et al.2013](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0061005).
##### Maize paralogous genes
Maize paralogous genes identified by [Schnable & Freeling 2011](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0017855).

#### This project
While trying to repeat the analysis listed above, we didn't use the exact same genotype data and GWAS model, for the following reasons:

1. The original paper [Wallele et al. 2014](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004845#pgen.1004845.s008) was published four years ago. Since then, many new genomic data have been published by the maize research community. 

2. The genotype data, which contains ~28.9 million SNPs and ∼800,000 copy-number variants (CNVs), as well as the GWAS model is too large for our computation resources.

Thus, we used the following methods/resources to conduct the analysis:
##### Phenotype data
Same as the original paper

##### Genotype data
~2.1 millions SNPs from [Kusmec et al. 2017](https://www.nature.com/articles/s41477-017-0007-7)generated by RNA-Seq, also merged with HapMap1 and HapMap2.

##### GWAS model
[FarmCPUpp](https://github.com/amkusmec/FarmCPUpp) A C++ implementation of [FarmCPU model](http://www.zzlab.net/FarmCPU/)(Fixed and random model Circulating Probability Unification ).The GWAS model and softawre is effiecient for large-scale GWAS. Bonferroni correction (p<0.05) was used as cutoff for significant SNPs. 

##### Gene Expression data

Maize gene atlas published in [Walley et al. 2016](http://science.sciencemag.org/content/353/6301/814). Compared to the [Sekhon et al.2013](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0061005) used in the original study, this one has overall higher quality.

##### Maize paralogous genes
Maize paralogous genes identified by [Zhang et al. 2017](http://www.plantcell.org/content/29/8/1938). Compared to [Schnable & Freeling 2011](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0017855) in the original paper, this one has several major improvement (by the lab of the author of the [Schnable & Freeling 2011](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0017855) paper).
